android/src/main/java/io/invertase/firebase/notifications/RNFirebaseBackgroundNotificationsService.java

```diff
@@ -0,0 +1,32 @@
+package io.invertase.firebase.notifications;
+
+import android.content.Intent;
+import android.os.Bundle;
+import android.util.Log;
+
+import com.facebook.react.HeadlessJsTaskService;
+import com.facebook.react.bridge.Arguments;
+import com.facebook.react.bridge.WritableMap;
+import com.facebook.react.jstasks.HeadlessJsTaskConfig;
+import com.google.firebase.messaging.RemoteMessage;
+
+import javax.annotation.Nullable;
+
+import io.invertase.firebase.messaging.MessagingSerializer;
+
+public class RNFirebaseBackgroundNotificationsService extends HeadlessJsTaskService {
+  @Override
+  protected @Nullable HeadlessJsTaskConfig getTaskConfig(Intent intent) {
+    Bundle extras = intent.getExtras();
+    if (extras != null) {
+      WritableMap notificationMap = Arguments.makeNativeMap(intent.getExtras());
+      return new HeadlessJsTaskConfig(
+        "RNFirebaseBackgroundNotification",
```

Comment:
Can we change this to `RNFirebaseBackgroundNotificationAction`?

Reply:
Done